app/services/analytics_events.rb

```diff
@@ -839,19 +854,7 @@ def idv_doc_auth_submitted_pii_validation(
     )
   end
 
-  # The "hybrid handoff" step: Desktop user has submitted their choice to
-  # either continue via desktop ("document_capture" destination) or switch
-  # to mobile phone ("send_link" destination) to perform document upload.
-  # Mobile users still log this event but with skip_upload_step = true
-  def idv_doc_auth_upload_submitted(**extra)
-    track_event('IdV: doc auth upload submitted', **extra)
-  end
-
-  # Desktop user has reached the above "hybrid handoff" view
-  def idv_doc_auth_upload_visited(**extra)
-    track_event('IdV: doc auth upload visited', **extra)
-  end
-
+  # @identity.idp.previous_event_name IdV: doc auth optional verify_wait submitted
```

Comment:
I removed this line in a recent PR. It must have crept back in during a merge and needs to be removed again. I looked into the history, and as far as I can tell, there is no association between verify_wait and verify_proofing_results.

Reply:
Removed.